### ISSUE TITLE
Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://api.flickr.com/services/...

### ISSUE THREAD

**Issue by airmanopus** [airmanopus/drivinandflyin#87]
If this request fails, we must gracefully recover and just not display the picture of the aircraft.

https://app.getsentry.com/steve-noonan-associates/drivin-and-flyin/group/73496747/

```
NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'https://api.flickr.com/services/rest/?…
  at getAircraftImage (/drivinandflyin/js/flickr.min.js:1:997)
  at XMLHttpRequest.flightTrackTransferComplete (/drivinandflyin/js/drivinandflyin.js:558:31)
```

**Comment by airmanopus** [comment]
This is the response we get when there are no available photos for the requested aircraft:
```json
{"photos":{"page":1,"pages":0,"perpage":100,"total":"0","photo":[]},"stat":"ok"}
```

**Comment by airmanopus** [comment]
This is fixed, see #99
